public/static/js/patch_new.js

```diff
@@ -14,6 +14,7 @@ mciModule.controller('PatchController', function($scope, $filter, $window, notif
   // in the left panel. Also accounts for special toggle behavior when the user
   // is holding shift/meta/ctrl when clicking.
   $scope.selectVariant = function($event, index){
+    $scope.variant = $scope.variants[index]
```

Comment:
Do you do any sort of binding to this? If not I would use a smaller scope for this